frontend/src/components/common/LookupCodeDropdown.test.tsx

```diff
@@ -0,0 +1,69 @@
+import * as MOCK from 'mocks/filterDataMock';
```

Comment:
I'd like to see a test in here for the 'all' functionality, but that doesn't need to be in this PR. I'll make a note of this.